tools/k8s/ci_cluster/cluster_connect.sh

```diff
@@ -0,0 +1,38 @@
+BASTION_INSTANCE_NAME=gke-circleci-bastion-newnet
+BASTION_INSTANCE_ZONE="us-west1-b"
+GCP_PROJECT_NAME=determined-ai
+    
+# Generate random number for remote port forwarding from the bastion instance to local host.
+MASTER_PORT_FROM_INTERNAL=$(jot -r 1 2000 65000)
```

Comment:
The `jot` binary isn't universally available, and I'd generally suggest starting higher than 2000.  Bash (and zsh) has a $RANDOM value which returns numbers from 0 to 32767 (15 bits). So perhaps...
```suggestion
MAX_PORT=65535
MIN_PORT=14000
MASTER_PORT_FROM_INTERNAL=$(( MIN_PORT + ((RANDOM + RANDOM) % (MAX_PORT - MIN_PORT)) ))
```
Ideally the port should still be checked for availability, and you normally do that by just opening an ephemeral port and returning whatever's opened. But for the purpose this is intended... 🤷‍♂️

Reply:
I ended up removing this since this would allow devs to launch workloads on the cluster, which isn't what we want in the case of the CI cluster.